### ISSUE TITLE
global.tls.functionsWorker.enabled is ignored

### ISSUE THREAD

**Issue by pgier** [datastax/kaap#123]
The functionsWorker tls setting seems to be ignored.  When I have `tls` enabled globally, but disabled on the functionsWorker, then the functionsWorker still gets TLS settings activated.
```
  global:
    tls:
      enabled: true
      functionsWorker:
        enabled: false
```

**Comment by dave2wave** [comment]
Please feel free to comment on the work with #110 and if something else is needed in that fix.

**Comment by nicoloboschi** [comment]
@pgier what are the TLS settings that you should expect to be disabled ?

**Comment by pgier** [comment]
@nicoloboschi The documentation isn't that clear about the behavior of the `global.tls.enabled` setting.  It seems that this is a global override and not a global default.  I think if we can clarify this in the docs, then we can close this issue.

**Comment by pgier** [comment]
I think the confusion here was that enabling TLS on the broker causes the functionsWorker to also require mounting the TLS certificate.  I'm going to close this one because it should be resolved by #135
